documentation/add/10_single_filebacked_operations.md

```diff
@@ -0,0 +1,77 @@
+# Intermediaries of Single Filebacked Operations
+Date: 2023-01-11
+
+## Objectives and problems to solve
+
+Currently the implementation of single-filebacked objects is based upon a temporary file beinhg created as infrequently
+as possible and modifcations to the sqw data taking place within that file. The issue with this mechanism is that while
+it may result in less data being written if operations only modify components of the data (e.g. signal/variance for many
+unary/binary ops) if a job fails or is cancelled partway through the user is left without a viable (or worse yet a
+viable but incorrect/invalid object).
+
+Thus, it was decided that the operation should only replace the temporary file once the full operation has been
+completed and otherwise leaves behind a correct file and any intermediaries are cleared upon finalisation of an
```

Comment:
```suggestion
completed and otherwise leaves behind a correct file, and any intermediaries are cleared upon finalisation of an
```

Avoids ambiguity that `and any` goes with `correct file`.